MRN-Code/coins-logon-widget
# Add logged in check

Asana task: https://app.asana.com/0/37131652076106/75626335776412
- **Problem:** Users see the login form, even if they’ve already logged in.
- **Solution:** Add an authentication check and show the “logged in” state if the user is logged in.
- **Testing:**
  - Run `grunt test`: all tests should be :+1:
  - Upload this to a public location on a _mrn.org_:
  
  ``` html
  <!doctype html>
  <html>
      <head>
          <title>COINS Logon Widget: Logged In Test</title>
          <link rel="stylesheet" href="//devcoin1.mrn.org/coins-logon-widget/dist/coins-logon-widget.min.css">
      <body>
          <div id="test"></div>
  
          <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js"></script>
          <script src="//devcoin1.mrn.org/coins-logon-widget/dist/coins-logon-widget.js"></script>
          <script>
              var myWidget = new CoinsLogonWidget({
                  el: document.getElementById('test'),
                  baseUrl: 'https://coins-api.mrn.org/api/v1.3.0',
                  authCookieName: 'CAS_Auth_User',
              });
          </script>
      </body>
  </html>
  ```
  
  Visit the page, log in, then hard-refresh. You should see the logged in widget state:
  
  ![logged-in-magic](https://cloud.githubusercontent.com/assets/1858316/11944180/0fddc10a-a7f9-11e5-864d-691d19bfa3c5.gif)
